lua/ide/lib/sort.lua

```diff
@@ -0,0 +1,100 @@
+-- Modified from the MIT-licensed code
```

Comment:
this file is copy/pasted from one of my own plugins -- let me know if you'd like me to reformat.

I'd recommend adding a `.stylua.toml` config to the repo and in a separate PR, formatting everything at once with stylua.